cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/SqlTransactionRunner.java

```diff
@@ -42,18 +42,22 @@
   private final DataSource dataSource;
   private final MetricsCollectionService metricsCollectionService;
   private final boolean emitTimeMetrics;
+  private final int scanFetchSize;
 
   @VisibleForTesting
   public SqlTransactionRunner(StructuredTableAdmin admin, DataSource dataSource) {
-    this(admin, dataSource, new NoOpMetricsCollectionService(), false);
+    // Keeping the scan fetch size to 100 as this is only visible for testing and not being called in code.
```

Comment:
Where is it used? I did a brief check and did not find usage in test code. Overall it's an antipattern to have test-only code in prod code. It's better to have a factory method in test code if needed

Reply:
Not sure where this is used. Agreed with the anti-pattern part.

Reply:
Removed code.